dashboard/src/actions/kube.tsx

```diff
@@ -0,0 +1,39 @@
+import { ThunkAction } from "redux-thunk";
+import { ActionType, createAction } from "typesafe-actions";
+import { Kube } from "../shared/Kube";
+import { IKubeItem, IStoreState } from "../shared/types";
+
+export const requestResource = createAction("REQUEST_RESOURCE", resolve => {
+  return (resource: { [s: string]: IKubeItem }) => resolve(resource);
+});
+
+export const receiveResource = createAction("RECEIVE_RESOURCE", resolve => {
+  return (resource: { [s: string]: IKubeItem }) => resolve(resource);
+});
+
+export const errorKube = createAction("ERROR_KUBE", resolve => {
```

Comment:
Maybe we should rename this to something less cryptic and more specific? `ReceiveResourceError`?